test/addons-napi/test_general/test.js

```diff
@@ -34,3 +34,12 @@ assert.ok(test_general.testGetPrototype(baseObject) !==
 // test version management functions
 // expected version is currently 1
 assert.strictEqual(test_general.testGetVersion(), 1);
+
+[123, 'test string', function() {}, new Object(), true,
+ undefined, Symbol()].forEach((val) => {
+   assert.strictEqual(typeof val, typeof (test_general.testNapiTypeof(val)));
```

Comment:
Since this test is specifically for `napi_typeof()`, I don't understand why `test_general.testNapiTypeof()` returns a value of that type instead of just a string representing the type. Then, this assertion could be:

`assert.strictEqual(typeof val, test_general.testNapiTypeof(val));`